src/main/java/org/folio/linked/data/mapper/kafka/impl/BibframeAuthorityIdentifiersInnerMapper.java

```diff
@@ -0,0 +1,19 @@
+package org.folio.linked.data.mapper.kafka.impl;
+
+import org.folio.linked.data.domain.dto.InstanceResponse;
+import org.folio.search.domain.dto.BibframeAuthorityIdentifiersInner;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BibframeAuthorityIdentifiersInnerMapper extends
+  IndexIdentifierMapperImpl<BibframeAuthorityIdentifiersInner, BibframeAuthorityIdentifiersInner.TypeEnum> {
```

Comment:
May be we could get rid of second parametrization? What if we just write an abstract method returning a TypeEnum class?

Reply:
discussed